pkg/platform/local/platform.go

```diff
@@ -552,6 +552,27 @@ func (p *Platform) CreateProject(ctx context.Context, createProjectOptions *plat
 	return nil
 }
 
+func (p *Platform) GetFunctionProject(ctx context.Context, functionConfig *functionconfig.Config) (platform.Project, error) {
```

Comment:
How is this different from the same function in `kube`?
If it's not - move to abstract

Reply:
@TomerShor yes, it is completely the same. But we need to keep them in kube/local because they're using `GetProjects` function which is specific for each platform

Reply:
The abstract platform object has a `platform` property, which gives access to the specific platform (kube / local).
So you can move this function to abstract, and call `GetProjects` like this:
```
projects, err := ap.platform.GetProjects(...)
```

Reply:
@TomerShor great stuff, moved it to abstract